workspace-manager-clienttests/src/main/resources/suites/BasicResiliency.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "BasicResiliency",
   "description": "Resiliency tests",
-  "serverSpecificationFile": "workspace-wsmtest-resiliency.json",
+  "serverSpecificationFile": "workspace-wsmtest.json",
```

Comment:
optional nit: In TDR, I think we always set the default server to the local one. I found that helpful for debugging things locally, and GH actions always overrode the server via env var anyway.

Reply:
For the sake of consistency with respect to `BasicPerf.json` and `BasicIntegration.json` (both use `workspace-wsmtest`), I'll leave this up to developers to decide the default `ServerSpecification` file to use.